scipy/stats/_multivariate.py

```diff
@@ -1332,6 +1413,10 @@ def _standard_rvs(self, n, shape, dim, df):
             Dimension of the scale matrix
         df : int
             Degrees of freedom
+        random_state : None or int or np.random.RandomState instance, optional
+            If int or RandomState, use it for drawing the random variates.
+            If None (or np.random), the global np.random state is used.
+            Default is None.
```

Comment:
This docstring is not quite right here: the random_state parameter should be an actual RandomState, no? The int-or-None game is played at the level of the public RVs method.